### DISCUSSION TITLE
FPrime Contribution

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1373] [Q&A]
Hello,

How can one participate in the contribution to FPrime repository ? Does it allow the contribution from outside of NASA into this repository from curious coders.

**Comment by LeStarch** [comment]
Contribution guidelines are found here: https://github.com/nasa/fprime/blob/devel/CONTRIBUTING.md

Contributions are allowed outside of NASA, however; these contributions are check via CI and code review from the F´ team before merging.
